apps/client/src/features/rundown/event-block/composite/EventBlockReporter.tsx

```diff
@@ -0,0 +1,93 @@
+import { Tooltip } from '@chakra-ui/react';
+import { OntimeEvent, ReportData } from 'ontime-types';
+import { isPlaybackActive, MILLIS_PER_MINUTE, MILLIS_PER_SECOND } from 'ontime-utils';
+
+import { usePlayback, useTimeUntil } from '../../../../common/hooks/useSocket';
+import useReport from '../../../../common/hooks-query/useReport';
+import useRundown from '../../../../common/hooks-query/useRundown';
+import { cx } from '../../../../common/utils/styleUtils';
+import { formatDuration, getTimeToStart } from '../../../../common/utils/time';
+import { tooltipDelayFast } from '../../../../ontimeConfig';
+
+import style from './EventBlockReporter.module.scss';
+
+interface EventBlockReporterProps {
+  id: string;
+  timeStart: number;
+  isPast: boolean;
+  isLoaded: boolean;
+  className: string;
+}
+
+function EventReport(props: { id: string; className: string }) {
+  const { id, className } = props;
+  const { data } = useReport();
+  const {
+    data: { rundown },
+  } = useRundown();
+
+  const report: ReportData | undefined = data[id];
+
+  if (report && report.startAt !== null && report.endAt !== null) {
+    const thisEvent = rundown[id] as OntimeEvent;
```

Comment:
nit: the word `this` carries to much baggage in javascript
can we call it something else, maybe `currentEvent`?